pkg/controllermanager/controller/controllerinstallation/controllerinstallation_control.go

```diff
@@ -261,67 +261,30 @@ func (c *defaultControllerInstallationControl) reconcile(controllerInstallation
 	}
 	conditionValid = helper.UpdatedCondition(conditionValid, gardencorev1alpha1.ConditionTrue, "RegistrationValid", "Chart could be rendered successfully.")
 
-	var (
-		manifest        = release.Manifest()
-		newResources    DeployedResources
-		newResourcesSet = sets.NewString()
-
-		decoder    = yaml.NewYAMLOrJSONDecoder(bytes.NewReader(manifest), 1024)
-		decodedObj map[string]interface{}
-	)
-
-	for err = decoder.Decode(&decodedObj); err == nil; err = decoder.Decode(&decodedObj) {
-		if decodedObj == nil {
-			continue
-		}
-
-		newObj := unstructured.Unstructured{Object: decodedObj}
-		decodedObj = nil
-
-		objectReference := corev1.ObjectReference{
-			APIVersion: newObj.GetAPIVersion(),
-			Kind:       newObj.GetKind(),
-			Name:       newObj.GetName(),
-			Namespace:  newObj.GetNamespace(),
-		}
-		newResources.Resources = append(newResources.Resources, objectReference)
-		newResourcesSet.Insert(objectReferenceToString(objectReference))
+	// Create secret
+	data := make(map[string][]byte, len(release.Files()))
+	for fileName, fileContent := range release.Files() {
+		key := strings.Replace(fileName, "/", "_", -1)
+		data[key] = []byte(fileContent)
 	}
 
-	if err := c.cleanOldResources(k8sSeedClient, controllerInstallation, newResourcesSet); err != nil {
-		if isDeletionInProgressError(err) {
-			conditionInstalled = helper.UpdatedCondition(conditionInstalled, gardencorev1alpha1.ConditionFalse, "DeletionPending", err.Error())
-		} else {
-			conditionInstalled = helper.UpdatedCondition(conditionInstalled, gardencorev1alpha1.ConditionFalse, "DeletionFailed", fmt.Sprintf("Deletion of old resources failed: %+v", err))
-		}
+	secretName := "managedresource-" + controllerInstallation.Name
```

Comment:
Why does this have the `managedresource-` prefix and the MR itself does not?
Should we prefix both with `controllerinstallation-<installationName>` or simply keep the `<installationName>` as name for both secret and MR?

Reply:
I changed it to `<installationName>` for both mr and secret name.